tools-local/unix/tool-helper.sh

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bash
+
+# Provides helper functions.
+
+# Gets the repository root path.
+get_repo_root()
+{
+    scriptPath="$(cd "$(dirname "$0")"; pwd -P)"
+    repoRoot="$(cd "$scriptPath/../.."; pwd -P)"
```

Comment:
Hmm, this looks like it shouldn't work... since this script is dot-sourced rather than executed, I'd think `$0` would refer to the script that's running this function, not the helper file. If an extension script calls this, I think it would put `repoRoot` somewhere outside the repo.

I think we *might* actually want to pass in the repo root everywhere.